cumulusci/cli/cci.py

```diff
@@ -825,14 +844,19 @@ def callback(*args, **kwargs):
                 validator = import_global(validator_path)
                 validator(serv_conf)
 
-            runtime.keychain.set_service(name, ServiceConfig(serv_conf), project)
+            service_name = kwargs["service_name"]
+            runtime.keychain.set_service(
+                service_type, service_name, ServiceConfig(serv_conf), project
+            )
             if project:
-                click.echo(f"{name} is now configured for this project.")
+                click.echo(f"{service_type} is now configured for this project.")
```

Comment:
This is potentially misleading. We are now only using the project flag to determine where the service gets set as the default, and the newly connected service will only get set as the default if it is the first one of this service_type.

I think we might want to do this:
1. Add a new `--default` flag which determines whether to set the new service as the global default for its type.
2. Repurpose `--project` to mean that the new service should be set as the project-specific default for its type.
3. Set both of these defaults via separate calls to set_default_service, so set_service doesn't need the project flag.

Reply:
> 3\. Set both of these defaults via separate calls to set_default_service, so set_service doesn't need the project flag.

set_service would still need the project flag it is going to set a default for the first service of a given type that is added. I suppose that I could take that logic out and implment it in the cci command. So if after we add a given service, and we detect that it is the first of its kind, then we call set_default_service() in that case as well (only if  --default and --project 
are not specified)

Reply:
@Br4nd0R I think it's okay to keep the "set the first service of a type as the default for that type" logic in the keychain (and that seems preferable to make sure that it always happens). This implicit default should always be a global default, not project. And then the cci command can additionally use set_default_service to make it a project-level service if that's what the user asked for.